README.md

```diff
@@ -1,30 +1,201 @@
 # Haitaton 2.0
-Haitaton 2.0 backend project.
+
+Haitaton is a service owned by the city of Helsinki that supports the management and prediction of the adverse
+effects of projects taking place within the urban area.
 
 ## Requirements
 
-TODO: document versions (once we settle with them).
+* OpenJDK (version 17+)
+* Docker
+* [Docker-compose](https://docs.docker.com/compose/install/)
+* IntelliJ IDEA is recommended for development purposes
 
-Using Idea:
-* IntelliJ Idea
-   * it contains its own JDK, kotlinc, gradle, etc.
-* OpenJDK (version 17+) - for running things after they have been built
+## How to compile, build and run
 
-Manual build
-* OpenJDK (version 17+)
+Building the service with both unit and integration tests:
+
+```
+$ ./gradlew :services:hanke-service:clean :services:hanke-service:build :services:hanke-service:integrationTest
+```
 
-## Modules
+Starting the application/services can be done afterwards with command line at haitaton-backend root directory:
 
-* services/auth-service
-  * Haitaton 2.0 Authentication service
-* services/hanke-service
-  * Haitaton 2.0 Hanke service
+```
+$ ./gradlew :services:hanke-service:bootRun
+```
 
-## How to compile, build and run
+### Docker compose
 
-See README.md in each individual module. It is not meant to be built *everything* from the root but build each service on its own.
-But, it is still possible to build everything by running
+Required directory structure:
+
+``` 
+  ├── haitaton
+  │   ├── haitaton-backend
+  │   └── haitaton-ui
 ```
-$ ./gradlew build
+
+#### Usage examples
+
+| Command                                                                                 | Explanation                                                                      |
+|-----------------------------------------------------------------------------------------|----------------------------------------------------------------------------------|
+| `docker-compose up -d`                                                                  | Run the entire stack, -d can be omitted                                          |
+| `docker-compose up db haitaton-hanke`                                                   | Run individual services                                                          |
+| `docker-compose stop haitaton-ui && docker-compose up -d --build --no-deps haitaton-ui` | Rebuild and run an individual service                                            |
+| `docker-compose down`                                                                   | Stop containers and removes containers networks volumes and images created by up |
```

Comment:
I find this really annoying to read in the editor. It's way too wide. Maybe some other solution than a table would be better?

Reply:
I agree that it is annoying to read in the editor, but in the actual usage it looks pretty good to me.
![image](https://github.com/City-of-Helsinki/haitaton-backend/assets/78962935/d5a43879-44da-43e9-b2a2-a39b99e624a2)


I can try bullet points, if the are ok?

Reply:
I mostly look at the README in the editor, or otherwise like in GitHub diffs, as raw markdown. So I think both the markdown and the rendered view are important to be readable.

Maybe nested bullet points?
```
- `docker-compose up -d`
  - Run the entire stack, -d can be omitted
- `docker-compose up db haitaton-hanke`
  - Run individual services
- `docker-compose stop haitaton-ui && docker-compose up -d --build --no-deps haitaton-ui`
  - Rebuild and run an individual service
- `docker-compose down`
  - Stop containers and removes containers networks volumes and images created by up
```